examples/tutorials/flow-deploy/azure-app-service/deploy.ps1

```diff
@@ -22,9 +22,9 @@ The app subscription, default using az account subscription.
 verbose mode.
 
 .EXAMPLE
-PS> .\deploy.ps1 -Path <folder-path> -Name my_app_23d8m -i <image_tag> -r <registry> -n <app_name> -g <resource_group>
+PS> .\deploy.ps1 -Path <folder-path> -i <image_tag> -r <registry> -n <app_name> -g <resource_group>
 .EXAMPLE
-PS> .\deploy.ps1 -Path <folder-path> -Name my_app_23d8m -i <image_tag> -r <registry> -n <app_name> -g <resource_group> -Subscription "xxxx-xxxx-xxxx-xxxx-xxxx" -Verbose
+PS> .\deploy.ps1 -Path <folder-path> -i <image_tag> -r <registry> -n <app_name> -g <resource_group> -Subscription "xxxx-xxxx-xxxx-xxxx-xxxx" -Verbose
```

Comment:
why remove Name?

Reply:
in this example command, it seems "-n <app_name>" and "-Name my_app_23d8m " are same, maybe keeping one is enough?

Reply:
good catch.